<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="icon" type="image/png" href="https://static.igem.wiki/teams/5218/image/static/images/logo.png" />
    <title>PetalGlow</title>
    <!-- Reset and Custom Stylesheets -->
    <link rel="stylesheet" href="./static/css/reset.css">
    <link rel="stylesheet" href="./static/css/main.css">
    <link rel="stylesheet" href="./static/css/header.css">
    <link rel="stylesheet" href="./static/css/notebook.css">
</head>

<body>
    <!-- Header Section -->
    <header class="navbar">
        <!-- Logo -->
        <a class="navbar__logo" href="https://2024.igem.wiki/bgi-mammothedu-south/">
            <img class="navbar__logo-icon" src="https://static.igem.wiki/teams/5218/image/static/images/logo.png"
                width="80" alt="PetalGlow Logo" />
            <div class="navbar__logo-text">
                <div class="navbar__title" href="https://2024.igem.wiki/bgi-mammothedu-south/">Petalglow</div>
                <div class="navbar__subtitle" href="roster.html">BGI-MammothEdu-South</div>
            </div>
        </a>

        <!-- Toggle for mobile menu -->
        <input type="checkbox" id="menu-toggle" class="navbar__checkbox" />
        <label for="menu-toggle" class="navbar__toggle-btn">☰</label>

        <!-- Navigation Menu -->
        <nav class="navbar__menu">
            <!-- Home Link （Retrun to Home page） -->
            <a href="https://2024.igem.wiki/bgi-mammothedu-south/" class="navbar__menu-item">Home</a>

            <!-- Project Dropdown -->
            <div class="navbar__menu-item-dropdown">
                <a href="#" class="navbar__menu-item">Project</a>
                <div class="navbar__submenu">
                    <a href="./description.html" class="navbar__submenu-item">Description</a>
                    <a href="./engineering.html" class="navbar__submenu-item">Engineering</a>
                    <a href="./results" class="navbar__submenu-item">Results</a>
                    <a href="./results#plant" class="navbar__submenu-item">Plant</a>
                    <a href="./implementation" class="navbar__submenu-item">Implementation</a>
                    <a href="./contribution" class="navbar__submenu-item">Contribution</a>
                </div>
            </div>

            <!-- Wet Lab Dropdown -->
            <div class="navbar__menu-item-dropdown">
                <a href="#" class="navbar__menu-item">Wet Lab</a>
                <div class="navbar__submenu">
                    <a href="./parts.html" class="navbar__submenu-item">Parts</a>
                    <a href="./experiments" class="navbar__submenu-item">Experiments</a>
                    <a href="./notebook.html" class="navbar__submenu-item">Notebook</a>
                    <a href="./protocal" class="navbar__submenu-item">Protocal</a>
                    <a href="./safety" class="navbar__submenu-item">Safety</a>
                </div>
            </div>

            <!-- Dry Lab Dropdown -->
            <div class="navbar__menu-item-dropdown">
                <a href="#" class="navbar__menu-item">Dry Lab</a>
                <div class="navbar__submenu">
                    <a href="./model" class="navbar__submenu-item">Model</a>
                    <a href="./measurement" class="navbar__submenu-item">Measurement</a>
                </div>
            </div>

            <!-- Out Reach Dropdown -->
            <div class="navbar__menu-item-dropdown">
                <a href="#" class="navbar__menu-item">Out Reach</a>
                <div class="navbar__submenu">
                    <a href="./human-practices.html" class="navbar__submenu-item">Human Practices</a>
                    <a href="./education" class="navbar__submenu-item">Education</a>
                    <a href="./entrepreneurship" class="navbar__submenu-item">Entrepreneurship</a>
                    <a href="./inclusivity" class="navbar__submenu-item">Inclusivity</a>
                </div>
            </div>

            <!-- Team Dropdown -->
            <div class="navbar__menu-item-dropdown">
                <a href="#" class="navbar__menu-item">Team</a>
                <div class="navbar__submenu">
                    <a href="./roster.html" class="navbar__submenu-item">Roster</a>
                    <a href="./attributions.html" class="navbar__submenu-item">Attribution</a>
                    <a href="./partnership.html" class="navbar__submenu-item">Partnership</a>
                    <a href="./highlights.html" class="navbar__submenu-item">Highlights</a>
                    <a href="#" class="navbar__submenu-item">Medals</a>
                </div>
            </div>

            <!-- Close Button for mobile menu -->
            <label for="menu-toggle" class="navbar__close-btn">✖</label>
        </nav>
    </header>

    <!-- Preloader Section -->
    <div id="preloader">
        <div class="spinner"></div>
        <p>cooking...</p>
    </div>

    <section class="intro-section" id="intro-section">
        <!-- Navigation Dots for Carousel -->
        <div class="carousel-nav">
            <span class="active" onclick="showSlide(0)"></span>
            <span onclick="showSlide(1)"></span>
            <span onclick="showSlide(2)"></span>
        </div>

        <div class="intro-background">
            <h1>Results</h1>
        </div>
    </section>

    <!-- Main Section with Content and TOC -->
    <section class="description-page">
        <!-- Background Image for Content Section -->
        <div class="content-background">
            <!-- Table of Contents (Left Side) -->
            <nav class="toc">
                <ul>
                    <li><a href="#abstract">&#127849Abstract</a></li>
                    <li><a href="#basic">&#127849Basic</a></li>
                    <li><a href="#composite">&#127849Composite</a></li>
                    <li><a href="#function">&#127849Vector</a></li>
                    <li><a href="#plant">&#127849Plant</a></li>
                    <li><a href="#sensor">&#127849Bio sensor</a></li>
                    <li><a href="#references">&#127849References</a></li>
                </ul>
            </nav>

            <!-- Content Sections (Right Side) -->
            <div class="content" style="margin-top: -400px;">
                <section id="abstract">
                    <h3>Abstract</h3>
                    <div style="text-align: left;">
                        <ul style="list-style-type: disc;">
                            <li style="font-size: clamp(0.5rem, 1.25vw, 1rem); ">Research and Analysis of Basic Parts:
                                Conducted literature research to obtain the
                                target sequence and performed
                                protein structure alignment analysis.</li>
                            <li style="font-size: clamp(0.5rem, 1.25vw, 1rem); ">Construction of Composite Parts:
                                Successfully assembled 13 expression vectors and
                                created new composite expression
                                elements.</li>
                            <li style="font-size: clamp(0.5rem, 1.25vw, 1rem); ">Verification of Vector Function:
                                Successfully validated that TAL and C3H genes can
                                enhance brightness through
                                transient transformation of tobacco leaves. The GRF3-RtTAL and GRF3-AtC3H modules can
                                respond to formaldehyde signals.</li>
                            <li style="font-size: clamp(0.5rem, 1.25vw, 1rem); ">Plant Synthetic Biology: Evaluated the
                                impact of the plant growth cycle on the transient
                                transformation efficiency of
                                tobacco leaves and its applications in plant synthetic biology.</li>
                            <li style="font-size: clamp(0.5rem, 1.25vw, 1rem); ">Luminous Plant Sensors: Explored how
                                luminous plants can be developed into biosensors
                                and the feasibility of their
                                interaction with humans.</li>
                        </ul>
                        <p></p>
                    </div>
                </section>

                <section id="basic">
                    <h3>Analysis of Basic Parts</h3>
                    <div style="text-align: left;">
                        <p>We conducted computational analyses of C3H proteins to explore the relationship between their
                            structure and function.
                            Eight C3H sequences from six plant species were collected and analyzed for motif consensus,
                            tertiary structure, and
                            predicted active sites. The results indicated a high level of structural conservation across
                            the motifs and tertiary
                            structures of the C3H proteins (Figure 1 & Figure 2). Despite this overall structural
                            similarity, differences in light
                            intensity among the members were observed, leading to the hypothesis that slight structural
                            variations at the active
                            sites may affect catalytic activity.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/01.png" alt="s6">
                            <figcaption>
                                <p>Figure 1: Illustration of motif consensus labeled in respect to its locations on
                                    protein homologs.</p>
                            </figcaption>
                        </figure>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/02.png" alt="s6">
                            <figcaption>
                                <p>Figure 2: Illustration of the 8 C3H homologs in protein three-dimensional structure
                                    using the SWISS model.</p>
                            </figcaption>
                        </figure>
                        <p>The docking simulation results (Figure 3) highlighted key residues involved in binding the
                            substrate, p-coumaric acid.
                            Notably, residues such as R436 and G432 in At_CYP98A3 were conserved across homologs but
                            displayed subtle differences in
                            neighboring residues, which may contribute to variations in enzyme efficiency and
                            bioluminescence. These findings
                            provide valuable insights into how small structural differences at the active site can
                            influence the overall catalytic
                            performance of C3H proteins, laying the foundation for further research and optimization of
                            these enzymes for enhanced
                            bioluminescence.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/03.png" alt="s6">
                            <figcaption>
                                <p>Figure 3: Illustration of 5 C3H homologs in three dimensions with ligand P-coumaric
                                    acid binding to its specific active
                                    site, visualized by Pymol.</p>
                            </figcaption>
                        </figure>
                    </div>
                </section>

                <section id="composite">
                    <h3>Composite Parts Construction</h3>
                    <div style="text-align: left;">
                        <p>In three rounds of DBTL (Design-Build-Test-Learn), we successfully constructed 13 expression
                            vectors. Each recombinant
                            vector contains basic regulatory sequences, CDS sequences, and fused reporter genes. The
                            target gene sequences were
                            amplified using PCR cloning and the successful amplification was confirmed by agarose gel
                            electrophoresis, showing bands
                            of expected sizes. Using seamless cloning (or homologous recombination), these sequences
                            were then inserted into the
                            desired positions of the main vector, followed by transformation into E. coli TOP10 cells.
                            Single colonies were picked
                            for shaking culture, colony PCR, and agarose gel electrophoresis verification. The correct
                            single colony transformants
                            were confirmed by DNA sequencing.</p>
                        <p>The correct single colony transformants were then subjected to shaking culture, plasmid
                            extraction, and transformation
                            into competent Agrobacterium GV3101 cells. Single colonies were picked for shaking culture,
                            colony PCR, and agarose gel
                            electrophoresis verification to confirm the correct Agrobacterium single colony
                            transformants, which were then stored in
                            glycerol stock (culture: 50% sterile glycerol = 1:1, mixed and stored at -80°C).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/4.png" alt="s6">
                            <figcaption>
                                <p>Figure 1. Maps of the 13 successfully constructed expression vectors.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-GFP (pCAMBIA1300 backbone) vector backbone</b>
                            </p>
                        </div>
                        <p>In this project, the vector backbone for the insertion of the target gene CDS is the modified
                            pS1300-GFP, based on the
                            pCAMBIA1300 backbone. The insertion site is located in the multiple cloning site (MCS)
                            between the Super promoter and
                            GFP, specifically between Hind III and Kpn I (indicated by the green arrow in Figure 2A).
                        </p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/5.png" alt="s6">
                            <figcaption>
                                <p>Figure 2. Schematic diagram of the pS1300-GFP (pCAMBIA1300 backbone) vector backbone.
                                </p>
                                <p>A: Schematic of the pS1300-GFP
                                    vector backbone, with the green arrow indicating the insertion site for the target
                                    gene.</p>
                                <p>B: Expanded view of the vector
                                    sequence at the green arrow in A, showing the homologous arm junction sequences. The
                                    overhang of HA-F includes an added
                                    BamH I site, which is prepared for subsequent replacement of the Super promoter
                                    sequence.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-RtTAL-GFP vector construction</b>
                            </p>
                        </div>
                        <p>Figure 3 shows the process of amplifying the target gene RtTAL CDS (<a class="link"
                                href="http://parts.igem.org/Part:BBa_K5218005">BBa_K5218005</a>). Initially,
                            due to improper handling,
                            the desired band was not observed in the electrophoresis (Figure 3A). We then performed PCR
                            cloning again using the
                            plasmid as a template (Figure 3B). However, due to sample spillage on the floor during gel
                            loading, the recovered sample
                            quantity was insufficient. We used the PCR product as a template for cloning again and
                            obtained a new product (Figure
                            3C) with a length of 2206 bp. The re-amplified product resulted in non-specific bands, but
                            we decided to proceed with
                            direct purification and recovery. The double enzyme digestion of the plasmid GFP was
                            performed on this sample (with
                            extended digestion time followed by direct product purification). The recombinant single
                            colony E. coli verification
                            showed bands of different sizes. We selected the expected results for sequencing (Figure
                            3E), and the sequencing results
                            confirmed successful construction.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/6.png" alt="s6">
                            <figcaption>
                                <p>Figure 3: Results of pS1300-RtTAL-GFP vector construction</p>
                                <p>A: Agarose gel electrophoresis of RtTAL CDS PCR product. M represents the 1kb plus
                                    DNA marker, lane 1 shows the RtTAL
                                    CDS PCR product, lane 2 is circular plasmid GFP, lane 3 is linearized plasmid GFP
                                    double-digested with Hind III and Kpn
                                    I, and lane 4 is linearized plasmid GUS double-digested with Hind III and Kpn I. The
                                    product verification failed, as
                                    indicated in the figure.</p>
                                <p>B and C: Agarose gel electrophoresis of RtTAL CDS PCR products. M represents the 1kb
                                    plus DNA marker, lane 1 shows the
                                    RtTAL CDS PCR product. Amplification was successful, but due to template issues, the
                                    band morphology varied; therefore,
                                    the products were pooled for recovery and purification.</p>
                                <p>D: Single colonies of pS1300-RtTAL-GFP transformants on an LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into Escherichia coli TOP10, and colony growth appeared normal.</p>
                                <p>E: PCR validation of monoclonal E. coli cultures. M represents the 1kb plus DNA
                                    marker, lanes 1-12 show the PCR results
                                    for 12 tested single colonies. Lanes 6, 8, and 11 exhibit single bands at the
                                    correct position and were used for
                                    sequencing.</p>
                                <p>F: Sequencing results of the target gene RtTAL CDS. Although the sequence was not
                                    fully completed, it was considered
                                    correct due to the uniformity of the template.</p>
                                <p>G: Single colonies of pS1300-RtTAL-GFP transformants on an LB kanamycin+ rifampicin+
                                    plate. The extracted plasmid was
                                    transformed into Agrobacterium tumefaciens GV3101, and colony growth appeared
                                    normal.</p>
                                <p>H: PCR validation of monoclonal Agrobacterium cultures. M represents the 1kb plus
                                    DNA marker. Lanes 1-8 show the PCR
                                    results for eight tested single colonies, all of which show correct band
                                    positioning.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-RcTAL-GFP vector construction</b>
                            </p>
                        </div>
                        <p>Figure 4 shows the successful amplification of the target gene RcTAL CDS (<a class="link"
                                href="http://parts.igem.org/Part:BBa_K5218001">BBa_K5218001</a>),
                            with a length of 1657 bp. The
                            digested GFP product also appears higher than the circular control plasmid (as
                            linearized plasmids migrate more slowly
                            than circular plasmids during electrophoresis), indicating successful digestion (Figure
                            4A). The PCR validation of the
                            recombinant monoclonal colonies confirms that most of the monoclonal colonies were
                            successfully constructed, as verified
                            by PCR and sequencing (Figures 4C and 4D).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/7.png" alt="s6">
                            <figcaption>
                                <p>Figure 4: Results of pS1300-RcTAL-GFP vector construction</p>
                                <p>A: Agarose gel electrophoresis of RcTAL CDS PCR product. M represents the 1kb plus
                                    DNA marker, lane 1 shows the RcTAL
                                    CDS PCR product, lane 2 is circular plasmid GFP, and lane 3 is linearized plasmid
                                    GFP double-digested with Hind III and
                                    Kpn I. The results indicate successful amplification of RcTAL and successful
                                    digestion of GFP.</p>
                                <p>B: Single colonies of pS1300-RcTAL-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed into
                                    Escherichia coli TOP10, and colony growth appeared normal.</p>
                                <p>C: PCR validation of monoclonal E. coli cultures. M represents the 1kb plus DNA
                                    marker, lanes 1-22 show the PCR results
                                    for 22 tested single colonies. Bright bands with correct positions were observed,
                                    and randomly selected positive
                                    monoclonal colonies were used for sequencing.</p>
                                <p>D: Sequencing results of the target gene RcTAL CDS. The sequence was fully read, and
                                    the sequencing was successful.</p>
                                <p>E: Single colonies of pS1300-RcTAL-GFP transformants on LB kanamycin+ rifampicin+
                                    plate. The extracted plasmid was
                                    transformed into Agrobacterium tumefaciens GV3101, and colony growth appeared
                                    normal.</p>
                                <p>F: PCR validation of monoclonal Agrobacterium cultures. M represents the 1kb plus
                                    DNA marker, lanes 1-8 show the PCR
                                    results for eight tested single colonies with correct band positions. Lanes 9 and 10
                                    show the PCR results for empty
                                    vector pS1300-GFP, confirming the absence of RcTAL.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-NoRtTAL-GFP vector construction</b>
                            </p>
                        </div>
                        <p>Figure 5 shows the successful amplification of the target gene NoRtTAL CDS (<a class="link"
                                href="http://parts.igem.org/Part:BBa_K5218025">BBa_K5218025</a>),
                            with a length of 2206 bp. The
                            GFP digestion product is at the same position as the circular control plasmid, indicating
                            that an extended digestion
                            time is required before product purification (Figure 5A). PCR validation of the recombinant
                            monoclonal colonies confirms
                            that most of the monoclonal colonies were successfully constructed, as verified by PCR and
                            sequencing (Figures 5C and
                            5D).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/8.png" alt="s6">
                            <figcaption>
                                <p>Figure 5: Results of pS1300-NoRtTAL-GFP vector construction</p>
                                <p>A: Agarose gel electrophoresis of NoRtTAL CDS PCR product. M represents the 1kb plus
                                    DNA marker, lane 1 shows the
                                    successfully amplified NoRtTAL CDS PCR product, lane 2 is circular plasmid GFP, lane
                                    3 is linearized plasmid GFP
                                    double-digested with Hind III and Kpn I, and lane 4 is linearized plasmid GUS
                                    double-digested with Hind III and Kpn I.</p>
                                <p>B: Single colonies of pS1300-NoRtTAL-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into Escherichia coli TOP10, and colony growth appeared normal.</p>
                                <p>C: PCR validation of monoclonal E. coli cultures. M represents the 1kb plus DNA
                                    marker, lanes 1-23 show the PCR results
                                    for 23 tested single colonies. Bright bands with correct positions were observed,
                                    and randomly selected positive
                                    monoclonal colonies were used for sequencing.</p>
                                <p>D: Sequencing results of the target gene NoRtTAL CDS. Although the full sequence was
                                    not obtained, it was considered
                                    correct due to the uniformity of the template.</p>
                                <p>E: Single colonies of pS1300-NoRtTAL-GFP transformants on LB kanamycin+ rifampicin+
                                    plate. The extracted plasmid was
                                    transformed into Agrobacterium tumefaciens GV3101, and colony growth appeared
                                    normal.</p>
                                <p>F: PCR validation of monoclonal Agrobacterium cultures. M represents the 2000bp DNA
                                    marker, lanes 1-8 show the PCR
                                    results for eight tested single colonies, all of which show correct band
                                    positioning.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-AtC3H-GFP, pS1300-PpC3H-GFP vector construction</b>
                            </p>
                        </div>
                        <p>Figure 6 shows the successful amplification of the target gene AtC3H CDS (<a class="link"
                                href="http://parts.igem.org/Part:BBa_K5218002">BBa_K5218002</a>)
                            through two rounds of PCR.
                            The first round yielded a product with a length of 1525 bp, and the second round produced a
                            product with a length of
                            1536 bp. Similarly, PpC3H CDS (<a class="link"
                                href="http://parts.igem.org/Part:BBa_K5218007">BBa_K5218007</a>) was also successfully
                            amplified in two rounds,
                            with the first PCR product
                            measuring 1786 bp and the second 1601 bp (Figures 6A and 6B), resulting in products without
                            stop codons and with
                            recombinant overhangs.The PCR validation of monoclonal colonies for the recombinant
                            pS1300-AtC3H-GFP confirmed
                            that most of the colonies were
                            correctly constructed and verified through PCR and sequencing (Figures 6C, 6F, and 6G).
                            However, when constructing pS1300-PpC3H-GFP, colony PCR validation of the first connection
                            plates showed no bands
                            (Figure 6F, lanes 13-21), and multiple attempts to isolate colonies still resulted in no
                            bands. Seamless cloning was
                            repeated (Figure 6E), and subsequent colony PCR validation (Figure 6H) showed the expected
                            bands. Sequencing was
                            successful (Figure 6I).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/9.png" alt="s6">
                            <figcaption>
                                <p>Figure 6: Construction Results of pS1300-AtC3H-GFP and pS1300-PpC3H-GFP Vectors</p>
                                <p>A: First round of CDS PCR products with termination codons, without recombinant
                                    overhangs, visualized on agarose gel
                                    electrophoresis. Lane M: 1 kb plus DNA marker; Lane 1: AtC3H CDS PCR product; Lane
                                    2: PpC3H CDS PCR product. The
                                    results confirm successful amplification of AtC3H and PpC3H CDS.</p>
                                <p>B: Second round of CDS PCR products without termination codons and with recombinant
                                    overhangs, shown on agarose gel
                                    electrophoresis. Lane M: 1 kb plus DNA marker; Lane 1: AtC3H CDS PCR product; Lane
                                    2: PpC3H CDS PCR product. The
                                    results confirm successful amplification of AtC3H and PpC3H CDS.</p>
                                <p>C: Single colonies of pS1300-AtC3H-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into E. coli TOP10, and the plate showed normal colony growth.</p>
                                <p>D and E: Single colonies of pS1300-PpC3H-GFP transformants on LB kanamycin+ plate.
                                    The recombinant vectors were
                                    transformed into E. coli TOP10, and both plates showed normal colony growth.</p>
                                <p>F: Colony PCR validation of pS1300-AtC3H-GFP transformants. Lane M: 1 kb plus DNA
                                    marker; Lanes 1-12: PCR results of
                                    12 tested single colonies. Bright single bands at the correct positions indicate the
                                    correct transformants, which
                                    were randomly selected and confirmed by sequencing.</p>
                                <p>G: Sequencing results of the AtC3H CDS gene. The entire sequence was successfully
                                    obtained, confirming accurate
                                    sequencing.</p>
                                <p>H: Colony PCR validation of pS1300-PpC3H-GFP transformants. Lane M: 1 kb plus DNA
                                    marker; Lanes 1-24: PCR results of
                                    24 tested single colonies. Bright single bands at the correct positions indicate
                                    correct transformants, which were
                                    randomly selected and confirmed by sequencing.</p>
                                <p>I: Sequencing results of the PpC3H CDS gene. The entire sequence was successfully
                                    obtained, confirming accurate
                                    sequencing.</p>
                                <p>J: Single colonies of pS1300-AtC3H-GFP transformants on LB kanamycin+ rifampicin+
                                    plate. The extracted plasmid was
                                    transformed into Agrobacterium GV3101, and the plate showed normal colony growth.
                                </p>
                                <p>L: Colony PCR validation of single-clone Agrobacterium carrying pS1300-AtC3H-GFP.
                                    Lane M: 1 kb plus DNA marker; Lanes
                                    1-12: PCR results of 12 tested single colonies, showing bands at the correct
                                    positions.</p>
                                <p>M: Single colonies of pS1300-PpC3H-GFP transformants on LB kanamycin+ rifampicin+
                                    plate. The extracted plasmid was
                                    transformed into Agrobacterium GV3101, and the plate showed normal colony growth.
                                </p>
                                <p>K: Colony PCR validation of single-clone Agrobacterium carrying pS1300-PpC3H-GFP.
                                    Lane M: 1 kb plus DNA marker; Lanes
                                    1-23: PCR results of 23 tested single colonies, all showing bands at the correct
                                    positions.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-NtC3H-GFP vector construction</b>
                            </p>
                        </div>
                        <p>Figure 7 shows the successful two-round PCR amplification of the target gene NtC3H CDS
                            (<a class="link" href="http://parts.igem.org/Part:BBa_K5218011">BBa_K5218011</a>). The first
                            round of PCR produced a product with a length of 1782 bp, and the second round produced a
                            product with a length of 1536
                            bp (Figure 7A and Figure 7B), ultimately obtaining a product without a termination codon and
                            with recombinant overhangs.
                            PCR validation of the recombinant single colonies confirmed that most of the single colonies
                            were successfully
                            constructed, as verified by PCR and sequencing (Figure 7D and Figure 7E).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/10.png" alt="s6">
                            <figcaption>
                                <p>Figure 7: Construction Results of pS1300-NtC3H-GFP Vector</p>
                                <p>A: First round of NtC3H CDS PCR products (with termination codons, without
                                    recombinant overhangs) on agarose gel
                                    electrophoresis. M: 1 kb plus DNA marker; 1: successfully amplified NtC3H CDS PCR
                                    product.</p>
                                <p>B: Second round of NtC3H CDS PCR products (without termination codons, with
                                    recombinant overhangs) on agarose gel
                                    electrophoresis. M: 1 kb plus DNA marker; 1: successfully amplified NtC3H CDS PCR
                                    product.</p>
                                <p>C: Single colonies of pS1300-NtC3H-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into E. coli TOP10, showing normal colony growth.</p>
                                <p>D: Colony PCR validation of single-clone E. coli. M: 1 kb plus DNA marker; lanes
                                    1-12: PCR results of 12 tested
                                    single colonies; clear single bands at the correct positions. Correct single clones
                                    were randomly selected for
                                    sequencing validation.</p>
                                <p>E: Sequencing results of the target gene NtC3H CDS. The entire sequence was
                                    successfully obtained, confirming
                                    accurate sequencing.</p>
                                <p>F: Single colonies of pS1300-NtC3H-GFP transformants on LB kanamycin+ rifampicin+
                                    plate. The extracted plasmid was
                                    transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>G: Colony PCR validation of single-clone Agrobacterium. M: 1 kb plus DNA marker;
                                    lanes 1-8: bands at the correct
                                    positions for 8 tested colonies.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-SlC3H-GFP vector construction</b>
                            </p>
                        </div>
                        <p>Figure 8 shows the successful two-round PCR amplification of the target gene SlC3H CDS
                            (<a class="link" href="http://parts.igem.org/Part:BBa_K5218013">BBa_K5218013</a>). The first
                            round
                            of PCR produced a product with a length of 1573 bp, and the second round produced a product
                            with a length of 1545 bp
                            (Figure 8A and Figure 8B), ultimately obtaining a product without a termination codon and
                            with recombinant overhangs.
                            PCR validation of the recombinant single colonies confirmed that most of the single colonies
                            were successfully
                            constructed, as verified by PCR and sequencing (Figure 8D and Figure 8E).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/11.png" alt="s6">
                            <figcaption>
                                <p>Figure 8: Construction Results of pS1300-SlC3H-GFP Vector</p>
                                <p>A: First round of SlC3H CDS PCR products (with termination codons, without
                                    recombinant overhangs) on agarose gel
                                    electrophoresis. M: 1 kb plus DNA marker; 1: successfully amplified SlC3H CDS PCR
                                    product.</p>
                                <p>B: Second round of SlC3H CDS PCR products (without termination codons, with
                                    recombinant overhangs) on agarose gel
                                    electrophoresis. M: 1 kb plus DNA marker; 1: successfully amplified SlC3H CDS PCR
                                    product.</p>
                                <p>C: Single colonies of pS1300-SlC3H-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into E. coli TOP10, showing normal colony growth.</p>
                                <p>D: Colony PCR validation of single-clone E. coli. M: 1 kb plus DNA marker; lanes
                                    1-12: PCR results of 12 tested
                                    single colonies; clear single bands at the correct positions. Correct single clones
                                    were randomly selected for
                                    sequencing validation.</p>
                                <p>E: Sequencing results of the target gene SlC3H CDS. Sequencing successfully completed
                                    for several single clones; one
                                    single clone had a segment deletion and was discarded.</p>
                                <p>F: Single colonies of pS1300-SlC3H-GFP transformants on LB kanamycin+ rifampicin+
                                    plate. The correctly sequenced
                                    plasmid was transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>G: Colony PCR validation of single-clone Agrobacterium. M: 2000 bp DNA marker; lanes
                                    1-8: bands at the correct
                                    positions for 8 tested colonies.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-CrC3H-GFP vector construction</b>
                            </p>
                        </div>
                        <p>Figure 9 shows the successful two-round PCR amplification of the target gene CrC3H CDS
                            (<a class="link" href="http://parts.igem.org/Part:BBa_K5218009">BBa_K5218009</a>). The first
                            round
                            of PCR produced a product with a length of 1688 bp, and the second round produced a product
                            with a length of 1569 bp
                            (Figure 9A and Figure 9B), ultimately obtaining a product without a termination codon and
                            with recombinant overhangs.
                            PCR validation of the recombinant single colonies confirmed that most of the single colonies
                            were successfully
                            constructed, as verified by PCR and sequencing (Figure 9D and Figure 9E).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/12.png" alt="s6">
                            <figcaption>
                                <p>Figure 9: Construction Results of pS1300-CrC3H-GFP Vector</p>
                                <p>A: First round of CrC3H CDS PCR products (with termination codons, without
                                    recombinant overhangs) on agarose gel
                                    electrophoresis. M: 1 kb plus DNA marker; 1: circular plasmid GFP; 2: Hind III\Kpn I
                                    double enzyme digestion linear
                                    plasmid GFP; 3: CrC3H CDS PCR product. The results indicate successful amplification
                                    of CrC3H CDS and successful
                                    digestion of GFP.</p>
                                <p>B: Second round of CrC3H CDS PCR products (without termination codons, with
                                    recombinant overhangs) on agarose gel
                                    electrophoresis. M: 1 kb plus DNA marker; 1 and 2: successfully amplified CrC3H CDS
                                    PCR products; the CrC3H CDS in
                                    lane 2 was successfully amplified.</p>
                                <p>C: Single colonies of pS1300-CrC3H-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into E. coli TOP10, showing normal colony growth.</p>
                                <p>D: Colony PCR validation of single-clone E. coli. M: 1 kb plus DNA marker; lanes
                                    1-12: PCR results of 12 tested
                                    single colonies; clear single bands at the correct positions. Correct single clones
                                    were randomly selected for
                                    sequencing validation.</p>
                                <p>E: Sequencing results of the target gene CrC3H CDS. Sequencing was complete, but with
                                    some individual base mutations,
                                    which were considered acceptable due to template variability.</p>
                                <p>F: Single colonies of pS1300-CrC3H-GFP transformants on LB kanamycin+ rifampicin+
                                    plate. The correctly sequenced
                                    plasmid was transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>G: Colony PCR validation of single-clone Agrobacterium. M: 5000 bp DNA marker; lanes
                                    1-16: bands at the correct
                                    positions for 16 tested colonies.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">pS1300-GUS (pCAMBIA1300 backbone) vector backbone</b>
                            </p>
                        </div>
                        <p>The vector backbone used for the target gene promoter in this project is pS1300-GUS, an
                            improved version based on the
                            pCAMBIA1300 backbone. The insertion site replaces the Super promoter sequence between the
                            BamH I sites on both ends
                            (indicated by the blue arrow in Figure 10A).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/13.png" alt="s6">
                            <figcaption>
                                <p>Figure 10: Schematic Diagram of the pS1300-GUS (pCAMBIA1300 backbone) Vector Backbone
                                </p>
                                <p>A: Schematic of the pS1300-GUS vector backbone, with the blue arrow indicating the
                                    insertion site for the target
                                    gene.</p>
                                <p>B and C: Expanded schematic of the vector sequence at the green arrows in A, showing
                                    the homologous arm linker
                                    sequences.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">p1300-GRF3-GUS vector construction</b>
                            </p>
                        </div>
                        <p>Figure 11 shows the successful amplification of the target gene GRF3 promoter (<a
                                class="link" href="http://parts.igem.org/Part:BBa_K5218017">BBa_K5218017</a>),
                            with a length of 1786 bp.
                            The GUS digestion product appears slightly higher than the circular control plasmid,
                            indicating successful digestion
                            (Figure 11A). PCR validation and sequencing of the recombinant single colonies confirmed
                            successful vector construction
                            (Figure 11C and Figure 11D). During PCR validation of single colonies after transforming the
                            recombinant plasmid into
                            Agrobacterium GV3101, initial attempts failed to produce the expected bands due to
                            undetected PCR machine malfunctions,
                            leading to multiple primer changes. Ultimately, only three single colonies could be verified
                            (Figure 11G, blue box).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/14.png" alt="s6">
                            <figcaption>
                                <p>Figure 11: Construction Results of p1300-GRF3-GUS Vector</p>
                                <p>A: GRF3 pro PCR product on agarose gel electrophoresis. M: 5000 bp DNA marker; 1:
                                    circular plasmid GUS; 2: BamH I
                                    digested linear plasmid GUS; 3: GRF3 pro PCR product. The figure shows successful
                                    amplification of the GRF3 product
                                    and successful digestion of GUS.</p>
                                <p>B: Single colonies of p1300-GRF3-GUS transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into E. coli TOP10, showing normal colony growth.</p>
                                <p>C: Colony PCR validation of single-clone E. coli. M: 5000 bp DNA marker; lanes 1-16:
                                    PCR results of 16 tested single
                                    colonies; lanes 12 and 13 show clear single bands at the correct positions, which
                                    were used for sequencing.</p>
                                <p>D: Sequencing results of the target gene GRF3 pro. Sequencing was fully completed
                                    successfully.</p>
                                <p>E: Single colonies of p1300-GRF3-GUS transformants on LB kanamycin+ rifampicin+
                                    plate. The correctly sequenced
                                    plasmid was transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>F and G: PCR validation of recombinant single-clone Agrobacterium. M: 5000 bp DNA
                                    marker; lanes 1-8: bands for eight
                                    single colonies amplified with different primers. Due to a PCR instrument
                                    malfunction (identified in later
                                    experiments), only three single colonies were validated using primer GUSP-seq-F/R,
                                    showing bands at correct
                                    positions. Additionally, the left side of Figure F shows PCR results of empty vector
                                    pS1300-GUS.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">p1300-GS1-GUS vector construction</b>
                            </p>
                        </div>
                        <p>Figure 12 shows the successful amplification of the target gene GS1 promoter (<a class="link"
                                href="http://parts.igem.org/Part:BBa_K5218019">BBa_K5218019</a>),
                            with a length of 997 bp
                            (Figure 12A). PCR validation and sequencing of the recombinant single colonies confirmed
                            successful vector construction
                            (Figure 12C and Figure 12D). After transforming the recombinant plasmid into Agrobacterium
                            GV3101, single colony PCR
                            validation was also successful (Figure 12F).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/15.png" alt="s6">
                            <figcaption>
                                <p>Figure 12: Construction Results of p1300-GS1-GUS Vector</p>
                                <p>A: GS1 pro PCR product on agarose gel electrophoresis. M: 5000 bp DNA marker; 1 and
                                    2: GS1 pro PCR products; lane 2
                                    shows the successful amplification of GS1 pro.</p>
                                <p>B: Single colonies of p1300-GS1-GUS transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed into
                                    E. coli TOP10, showing normal colony growth.</p>
                                <p>C: Colony PCR validation of single-clone E. coli. M: 5000 bp DNA marker; lanes 1-16:
                                    PCR results of 16 tested single
                                    colonies; clear single bands at the correct positions. Correct single clones were
                                    randomly selected for sequencing
                                    validation.</p>
                                <p>D: Sequencing results of the target gene GS1 pro. Sequencing was fully completed
                                    successfully.</p>
                                <p>E: Single colonies of p1300-GS1-GUS transformants on LB kanamycin+ rifampicin+ plate.
                                    The correctly sequenced plasmid
                                    was transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>F: Colony PCR validation of single-clone Agrobacterium. M: 5000 bp DNA marker; lanes
                                    1-8: bands at the correct
                                    positions for 8 tested colonies.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">p1300-MDH1-GUS vector construction</b>
                            </p>
                        </div>
                        <p>Figure 13 shows the successful amplification of the target gene MDH1 promoter (<a
                                class="link" href="http://parts.igem.org/Part:BBa_K5218021">BBa_K5218021</a>),
                            with a length of 341 bp.
                            The GUS digestion was incomplete, resulting in two bands, indicating the need to increase
                            the digestion time before
                            product recovery (Figure 13A). PCR validation and sequencing of the recombinant single
                            colonies confirmed successful
                            vector construction (Figure 13C and Figure 13D). After transforming the recombinant plasmid
                            into Agrobacterium GV3101,
                            single colony PCR validation was also successful (Figure 13F).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/16.png" alt="s6">
                            <figcaption>
                                <p>Figure 13: Construction Results of p1300-MDH1-GUS Vector</p>
                                <p>A: MDH1 pro PCR product on agarose gel electrophoresis. M: 5000 bp DNA marker; 1 and
                                    2: successfully amplified MDH1
                                    pro PCR products.</p>
                                <p>B: Single colonies of p1300-MDH1-GUS transformants on LB kanamycin+ plate. The
                                    recombinant vector was transformed
                                    into E. coli TOP10, showing normal colony growth.</p>
                                <p>C: Colony PCR validation of single-clone E. coli. M: 5000 bp DNA marker; lanes 1-16:
                                    PCR results of 16 tested single
                                    colonies; clear single bands at the correct positions. Correct single clones were
                                    randomly selected for sequencing
                                    validation.</p>
                                <p>D: Sequencing results of the target gene MDH1 pro. Sequencing was fully completed
                                    successfully for some single
                                    clones; two clones did not align correctly and were discarded.</p>
                                <p>E: Single colonies of p1300-MDH1-GUS transformants on LB kanamycin+ rifampicin+
                                    plate. The correctly sequenced
                                    plasmid was transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>F: Colony PCR validation of single-clone Agrobacterium. M: 5000 bp DNA marker; lanes
                                    1-8: bands at the correct
                                    positions for 8 tested colonies.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">p1300-pGRF3-RtTAL-GFP, p1300-pGRF3-AtC3H-GFP vector
                                    construction</b>
                            </p>
                        </div>
                        <p>The Super promoter in pS1300-RtTAL-GFP and pS1300-AtC3H-GFP was replaced with the GRF3
                            promoter, still using BamH I for
                            single enzyme digestion to remove the Super promoter sequence. Since we used a seamless
                            cloning method, the PCR products
                            required corresponding homologous arm overhangs for recombination. The 5' end of the GRF3
                            sequence was connected to the
                            basic backbone of both vectors, in the same position as the GUS vector, so the homologous
                            overhangs were the same.
                            However, the 3' end was connected to the RtTAL and AtC3H sequences, respectively, so the
                            recombination overhangs were
                            specific to each gene. The figure shows the successful amplification of the GRF3 promoter
                            (Figure 14A, lanes 5 and 6).
                            After replacing the PCR machine, both E. coli and Agrobacterium single-colony PCR
                            validations confirmed successful
                            vector construction (Figure 14F-L).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/17.png" alt="s6">
                            <figcaption>
                                <p>Figure 14: Construction Results of p1300-pGRF3-RtTAL-GFP and p1300-pGRF3-AtC3H-GFP
                                    Vectors</p>
                                <p>A: GRF3 pro PCR products on agarose gel electrophoresis. M: 1 kb plus DNA marker;
                                    lanes 1 and 2: successfully
                                    amplified MDH1 pro PCR products; lanes 3 and 4: successfully amplified GS1 pro PCR
                                    products; lanes 5 and 6:
                                    successfully amplified GRF3 pro PCR products. Lane 1, 3, and 5 contain homologous
                                    arm overhangs for recombination
                                    with RtTAL, while lanes 2, 4, and 6 contain homologous arm overhangs for
                                    recombination with AtC3H.</p>
                                <p>B: Single colonies of p1300-pGRF3-RtTAL-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was
                                    transformed into E. coli TOP10, showing normal colony growth.</p>
                                <p>C: Single colonies of p1300-pGRF3-AtC3H-GFP transformants on LB kanamycin+ plate. The
                                    recombinant vector was
                                    transformed into E. coli TOP10, showing normal colony growth.</p>
                                <p>D: Single colonies of p1300-pGRF3-RtTAL-GFP transformants on LB kanamycin+
                                    rifampicin+ plate. The correctly sequenced
                                    plasmid was transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>E: Single colonies of p1300-pGRF3-AtC3H-GFP transformants on LB kanamycin+
                                    rifampicin+ plate. The correctly sequenced
                                    plasmid was transformed into Agrobacterium GV3101, showing normal colony growth.</p>
                                <p>F: Colony PCR validation of p1300-pGRF3-RtTAL-GFP single-clone E. coli. M: 5000 bp
                                    DNA marker; lanes 1-24: PCR
                                    results of 24 tested single colonies; clear single bands at the correct positions.
                                    Correct single clones were
                                    randomly selected for sequencing validation.</p>
                                <p>G: Sequencing results of the target gene MDH1 pro. Sequencing was fully completed
                                    successfully.</p>
                                <p>H: Colony PCR validation of p1300-pGRF3-AtC3H-GFP single-clone E. coli. M: 5000 bp
                                    DNA marker; lanes 1-24: PCR
                                    results of 24 tested single colonies; clear single bands at the correct positions.
                                    Correct single clones were
                                    randomly selected for sequencing validation.</p>
                                <p>I: Sequencing results of the target gene MDH1 pro. Sequencing was fully completed
                                    successfully.</p>
                                <p>J and K: Colony PCR validation of p1300-pGRF3-RtTAL-GFP single-clone Agrobacterium.
                                    M: 5000 bp DNA marker; lanes 1-8:
                                    bands at the correct positions for 8 tested colonies with different primers.</p>
                                <p>J and K: Colony PCR validation of p1300-pGRF3-AtC3H-GFP single-clone Agrobacterium.
                                    M: 5000 bp DNA marker; lanes 1-8:
                                    bands at the correct positions for 8 tested colonies.</p>
                            </figcaption>
                        </figure>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/18.png" alt="s6">
                            <figcaption>
                                <p>Figure 15: Our Daily Laboratory Operations</p>
                            </figcaption>
                        </figure>
                        <p>Additionally, we attempted to amplify other C3H genes from different plant materials, such as
                            RcC3H from rose and
                            other homologous C3H sequences from the same species. However, due to material variability
                            and growth cycle
                            limitations, we did not obtain the expected bands. The vector construction process was also
                            not very smooth, and we
                            encountered multiple failures, including operational errors, equipment malfunctions, and
                            unknown reasons.
                            Fortunately, through repeated attempts, we became more proficient in various experimental
                            methods and learned
                            important considerations for repeating experiments. You can refer to <a class="link"
                                href="https://2024.igem.wiki/bgi-mammothedu-south/experiments">this link</a> for further
                            details.</p>
                    </div>
                </section>

                <section id="function">
                    <h3>Vector Function Verification</h3>
                    <div style="text-align: left;">
                        <p>We successfully contacted Professor Du from the glowing plant research team at Zhejiang
                            University, who was willing
                            to provide us with the first-generation glowing vector FBP-22 (introducing FBP, including
                            LUZ, H3H, CPH, and HispS
                            genes). We collaborated with the Mammoth Education system to construct the transgenic
                            tobacco line Nicotiana
                            benthamiana FBP-22/FBP-T2. The initial transgenic self-luminous tobacco lines FBP-22/FBP-T2
                            were proven to be
                            self-luminous. However, the intensity of the luminescence was limited and did not meet our
                            practical application
                            goals. Our team focuses on enhancing plant luminescence and exploring further applications,
                            using the FBP-T2 tobacco
                            line as a control and genetic engineering material for the functional verification of TAL
                            and C3H in FBP.</p>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Transient expression of RtTAL and AtC3H to enhance
                                    self-luminescence brightness</b>
                            </p>
                        </div>
                        <p>We separately transformed Agrobacterium GV3101 strains with pS1300-GFP and pS1300-gene
                            CDS-GFP constructs, followed
                            by transient transformation of tobacco leaves through injection. After 48 hours, transgenic
                            tobacco leaves were
                            examined under a fluorescence microscope. The results showed GFP fluorescence signals from
                            the gene CDS-GFP fusion
                            proteins in the epidermal cells of tobacco leaves, indicating the successful engineering of
                            each pS1300-gene CDS-GFP
                            construct.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/19.png" alt="s6">
                            <figcaption>
                                <p>Figure 1: Fluorescence signals of each pS1300-gene CDS-GFP vector in tobacco leaves
                                </p>
                                <p>Note: Blank Control is the tobacco line FBP-T2 used as a blank control, while the
                                    others were injected with
                                    individual pS1300-gene CDS-GFP vectors.</p>
                            </figcaption>
                        </figure>
                        <p>We observed the phenotypes in a completely dark setting using a DSLR camera (parameters: ISO
                            6400, aperture F2,
                            shutter speed 30s). The results showed that the experimental groups transiently expressing
                            the 8 pS1300-gene CDS-GFP
                            constructs exhibited higher brightness compared to the Control (only injected vector
                            pS1300-GFP) and Blank Control,
                            aligning with our expectations. This preliminary result indicates that the successfully
                            constructed TAL and C3H
                            genes can enhance luminescence intensity.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/20.png" alt="s6">
                            <figcaption>
                                <p>Figure 2: Self-luminescence signals of each pS1300-gene CDS-GFP vector in tobacco
                                    leaves</p>
                                <p>Note: Blank Control is the blank control of the tobacco line FBP-T2, while the others
                                    were injected with individual
                                    pS1300-gene CDS-GFP vectors. The left image shows phenotypic observations taken with
                                    a DSLR camera (ISO 6400,
                                    aperture F2, shutter speed 30s) under white light and in darkness 48 hours after
                                    injection. The plants, from left to
                                    right, are: Blank Control, Control (only injected vector), and pS1300-gene CDS-GFP.
                                    The table on the right shows the
                                    luminescence brightness data analyzed by ImageJ grayscale values.</p>
                            </figcaption>
                        </figure>
                        <p>To visually demonstrate that our constructs can enhance the self-luminescence brightness of
                            the first-generation
                            tobacco line FBP-T2, we performed uniform dotting on the injection areas and analyzed
                            luminescence brightness using
                            ImageJ software by measuring grayscale values. The results showed that these TAL and C3H
                            genes effectively enhanced
                            luminescence intensity. The C3H group consistently exhibited higher brightness than the
                            Control (only injected
                            vector), indicating that the C3H gene can enhance luminescence intensity. The highest
                            brightness of AtC3H reached a
                            grayscale value of 49, indicating that the AtC3H gene performed best in tobacco. Similarly,
                            the TAL group enhanced
                            luminescence intensity, with RtTAL achieving the highest brightness of 51 grayscale value,
                            indicating that the RtTAL
                            gene performed best in tobacco. Among the constructs, pS1300-RtTAL-GFP showed the highest
                            brightness peak after
                            injection, followed by pS1300-AtC3H-GFP, further validating the phenotypic results.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/engineering/150.png"
                                alt="s6">
                            <figcaption>
                                <p>Figure 3: light intensity of bioluminescence in different constructs transiently
                                    expressed in tobacco leaves</p>
                                <p>Note: The light intensity was calculated based on grayscale value with ImageJ
                                    software. 4 spots (radius 15pixels) were
                                    randomly selected on the photos, error bar represent standard deviation calculated
                                    from grayscale value.</p>
                            </figcaption>
                        </figure>
                        <p>In addition to data comparison and analysis, we also analyzed the variability of the data. As
                            shown in the figure
                            below, in the C3H group, AtC3H had a relatively higher peak and shorter length, with a more
                            concentrated
                            distribution, indicating the most significant growth in AtC3H. In the TAL group, RtTAL also
                            showed a relatively
                            higher peak and shorter length with a dense distribution, demonstrating that RtTAL had the
                            most growth. This
                            confirms that the data reflected in the previous figure are generally accurate, and that
                            AtC3H in the C3H group and
                            RtTAL in the TAL group are more densely clustered, indicating more stable transformation
                            repeatability.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/engineering/160.png"
                                alt="s6">
                            <figcaption>
                                <p>Figure 4: The data in this table are derived from the experimental group minus the
                                    Control(only injected vector) group</p>
                                <p>Note: The TAL group was significantly higher than the Control (only injected vector)
                                    group, with RtTAL showing the
                                    highest brightness, indicating that the RtTAL gene had the best expression
                                    performance in tobacco, followed by the
                                    AtC3H gene. The Box Plot is used to display the central tendency and dispersion of
                                    the data, where the interquartile
                                    range (IQR) is represented by the box, the median is indicated by a horizontal line,
                                    and the arithmetic mean of all
                                    data points is marked with an "x."</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Formaldehyde-Responsive Promoter-GUS Staining Test</b>
                            </p>
                        </div>
                        <p>Additionally, as glowing plant pets, our focus is on whether they can respond as switches to
                            emotional gases secreted
                            by humans in the environment. Due to limitations in Human Subjects Research Policy for
                            ethical safety and the
                            difficulty of human-related experiments, we chose formaldehyde, a common harmful gas in the
                            environment, as a
                            controllable variable parameter in our experiments.</p>
                        <p>We conducted transient infection and transformation of tobacco leaves using the blank control
                            (0.5x PBS), positive
                            control (pS1300-GUS), and experimental group (p1300-promoter-GUS constructs). After 12 hours
                            of injection, the H2O
                            control and 2 mM HCHO treatment were sprayed under the same conditions and cultured in a
                            sealed environment for 36
                            hours. Samples were taken, and GUS staining and decolorization were performed at 48 hours.
                        </p>
                        <p>Results showed that in the blank control group (0.5x PBS), neither H2O nor formaldehyde
                            spraying induced any GUS
                            signal. In the positive control group (pS1300-GUS), both H2O and formaldehyde treatments
                            resulted in deep blue GUS
                            signals, with formaldehyde treatment showing an even deeper blue. In the experimental group
                            (p1300-MDH1-GUS), GUS
                            signals were observed with both H2O and formaldehyde, but the blue color was weaker after
                            formaldehyde treatment,
                            and GUS signals were weaker than those in the GUS control group, indicating that
                            formaldehyde treatment inhibited
                            MDH1 promoter expression, though not significantly. In the experimental group
                            (p1300-GS1-GUS), blue signals appeared
                            after H2O treatment, but almost no GUS expression was observed after formaldehyde treatment,
                            suggesting that GS1
                            promoter expression was inhibited by formaldehyde. In the experimental group
                            (p1300-GRF3-GUS), blue signals were
                            visible after H2O treatment, but no GUS expression occurred after formaldehyde treatment,
                            showing that GRF3 promoter
                            expression was significantly inhibited by formaldehyde.</p>
                        <p>Overall, the promoter-GUS modules showed significant phenotypic responses to formaldehyde.
                            After formaldehyde
                            treatment, the promoters' expressions were inhibited to varying degrees. The p1300-GRF3-GUS
                            treatment exhibited the
                            most notable difference, with the GRF3 promoter completely inhibiting GUS gene expression in
                            response to
                            formaldehyde, resulting in no blue-stained tissues in the transformed leaves.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/engineering/250.png"
                                alt="s6">
                            <figcaption>
                                <p>Figure 5: Formaldehyde-Responsive Promoter-GUS Staining Results</p>
                                <p>Note: Tobacco leaves were transiently transformed using Agrobacterium-mediated
                                    transformation technology. After
                                    injection, 2 mM HCHO and blank control H2O were sprayed starting at 12 hours, with
                                    reagents replaced every 12 hours
                                    for a total of three treatments over 36 hours. At 48 hours post-injection, the
                                    injection areas were sampled in a
                                    fume hood, and the samples were stained overnight at 37°C and 75 rpm using the
                                    substrate X-Gluc. The next day, the
                                    samples were completely decolorized with absolute ethanol and placed in 9 cm Petri
                                    dishes for white-field
                                    photography.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Building a Controllable Luminescence Intensity Regulation
                                    Module</b>
                            </p>
                        </div>
                        <p>Next, we used the GRF3 promoter, which showed the most significant phenotypic
                            differences, to replace the Super
                            promoter sequence in the pS1300-RtTAL-GFP and pS1300-AtC3H-GFP vectors, creating
                            constructs p1300-pGRF3-RtTAL-GFP
                            (<a class="link" href="http://parts.igem.org/Part:BBa_K5218023">BBa_K5218023</a>) and
                            p1300-pGRF3-AtC3H-GFP (<a class="link"
                                href="http://parts.igem.org/Part:BBa_K5218024">BBa_K5218024</a>). This combined the
                            formaldehyde-responsive regulation module
                            with luminescence-enhancing genes to develop formaldehyde-responsive glowing plants.</p>
                        <p>The results after 36 hours of treatment with 2 mM HCHO showed that the control group
                            (pS1300-GFP) expressed normal
                            luminescence under water treatment, but luminescence was significantly suppressed under
                            formaldehyde treatment,
                            becoming very faint. In the experimental group p1300-pGRF3-RtTAL-GFP, where the Super
                            promoter was replaced with the
                            GRF3 promoter, normal luminescence was observed after water treatment, with brightness
                            significantly higher than
                            that of the GFP control group, and the brightness was almost unaffected by formaldehyde
                            treatment. In the
                            pS1300-RtTAL-GFP control group, the brightness did not change significantly after either
                            water or formaldehyde
                            treatment, though luminescence slightly weakened under formaldehyde, and overall
                            brightness was higher than in the
                            GFP group. This indicates that the GRF3 promoter responds to formaldehyde to some
                            extent, but the luminescence
                            difference is small, making it difficult to draw further conclusions.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/engineering/280.png"
                                alt="s6">
                            <figcaption>
                                <p>Figure 6: Formaldehyde-Responsive Results of p1300-pGRF3-RtTAL-GFP</p>
                                <p>Note: Using Agrobacterium-mediated transient transformation of tobacco leaves, 2 mM
                                    HCHO and blank control H2O were
                                    sprayed starting at 12 hours post-injection, with reagent replacement every 12 hours
                                    for a total of three treatments
                                    over 36 hours. At 48 hours post-injection, phenotypic observations were taken in
                                    both white light and darkness using
                                    a DSLR camera with ISO 6400, aperture F2, and shutter speed 30s.</p>
                            </figcaption>
                        </figure>
                        <p>In the formaldehyde response experiment of the p1300-pGRF3-RtTAL-GFP vector, the tobacco
                            seedlings used were at an
                            incorrect growth stage, making the injection of the infection solution difficult and
                            resulting in irregular
                            luminescent areas and significant leaf damage. This greatly affected the brightness analysis
                            and visual perception
                            of the results. Therefore, the results are for reference only, and after reflecting on the
                            reasons, we conducted a
                            more comprehensive second functional verification.</p>
                        <p>We observed a similar pattern in the p1300-pGRF3-AtC3H-GFP experiments as seen in the
                            pGRF3-RtTAL module. To better
                            contrast this phenomenon, we adopted a different photography method. As shown in the figure
                            below, the pS1300-GFP
                            control group was significantly inhibited by formaldehyde, resulting in extremely faint
                            luminescence. In the
                            p1300-pGRF3-AtC3H-GFP experimental group, luminescence slightly increased after formaldehyde
                            treatment compared to
                            water treatment. The brightness of the pS1300-AtC3H-GFP control group was generally
                            consistent, with formaldehyde
                            treatment resulting in slightly lower brightness than the p1300-pGRF3-AtC3H-GFP group.</p>
                        <p>Overall, the GRF3 promoter can respond to formaldehyde, enhancing the luminescence of glowing
                            plants, though the
                            effect is not very pronounced. Additionally, the AtC3H gene allows plants to maintain
                            near-normal luminescence under
                            formaldehyde inhibition, demonstrating the gene's stability. The results of this experiment
                            were improved based on
                            verification from different batches, using more mature tobacco plants and directly
                            photographing the luminescent
                            areas of the injected leaves for clearer results.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/engineering/290.png"
                                alt="s6">
                            <figcaption>
                                <p>Figure 7: Formaldehyde-Responsive Results of p1300-pGRF3-AtC3H-GFP</p>
                                <p>Note: The experimental procedure was the same as that used for the
                                    p1300-pGRF3-RtTAL-GFP construct. The injected leaves
                                    were photographed in 3D, with camera settings: ISO 6400, aperture F2, shutter speed
                                    30s.</p>
                            </figcaption>
                        </figure>
                        <p>Excitingly, the results from all the above functional validation experiments demonstrate that
                            both TAL and C3H genes
                            exhibit high luminescence stability. From the transient transformation verification of the
                            eight pS1300-gene CDS-GFP
                            constructs in tobacco leaves, we confirmed that the RtTAL and AtC3H genes effectively
                            enhance the self-luminescence
                            brightness of the first-generation transgenic tobacco, with grayscale values more than
                            doubling compared to the
                            control. The GRF3 promoter effectively responded to formaldehyde, meeting our expectations
                            and successfully creating
                            formaldehyde-responsive glowing plants. This experiment provides a promising validation of
                            the feasibility of
                            detecting gases released by human emotional fluctuations in the future.</p>
                        <p>Since the formaldehyde used in the experiments can cause some harm to plants, it is important
                            to note that the
                            overall physiological activity of synthetic biology within plants is inherently complex. GUS
                            staining is commonly
                            used to study gene promoter expression in plants, and we attempted to apply it to explore
                            the effects of
                            environmental factors on gene expression. The experiential analysis of the results can be
                            referenced <a class="link"
                                href="https://2024.igem.wiki/bgi-mammothedu-south/engineering.html">here</a>.</p>
                    </div>
                </section>

                <section id="plant">
                    <h3>Plant Synthetic Biology</h3>
                    <div style="text-align: left;">
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Introduction and Significance of Synthetic Biology</b>
                            </p>
                        </div>
                        <p>Synthetic biology, simply put, starts with the most basic elements of life, step by step
                            constructing parts and
                            components, and using the principles of modern engineering to synthesize artificial
                            organisms with specific functions
                            based on the principles of biological composition. It aims to create artificial
                            biological systems that can function
                            like circuits. Synthetic biology applies engineering concepts, using existing biological
                            components or modules to design
                            and construct biological systems with specific biological functions. The luminescent
                            systems found in glowing organisms,
                            especially the biological components related to light-emitting reactions, provide
                            valuable materials for the
                            modification of organisms. These components from luminous organisms are used as
                            biosensors or reporter systems for
                            monitoring the environment or scientific research, or for constructing glowing plants as
                            plant pets—an idea we are
                            exploring.</p>
                        <p>Glowing plants represent a fascinating area within synthetic biology, combining knowledge
                            from biology, genetic
                            engineering, and botany to create organisms with new functions. By using synthetic
                            biology approaches, researchers can
                            introduce foreign genes into plants, enabling them to express luminescent proteins and
                            thus granting the plants the
                            ability to glow.</p>
                        <p>1. Luminescence Mechanism: In synthetic biology, researchers typically utilize
                            luminescent genes extracted from other
                            organisms, such as fireflies, bioluminescent mushrooms, or marine organisms. These genes
                            encode proteins that produce
                            luminescence, such as luciferase or similar substances. Through genetic engineering
                            techniques, these genes can be
                            introduced into a plant’s genome, allowing the plant to glow under specific conditions.
                        </p>
                        <p>2. Applications and Significance: Glowing plants are not just scientific achievements;
                            they also hold potential for
                            practical applications. For example, glowing plants can act as environmental sensors,
                            detecting heavy metals,
                            pollutants, or pathogenic microorganisms in the soil, or even harmful gases.
                            Additionally, they can be used for
                            landscape beautification or as green lighting solutions.</p>
                        <p>3. Challenges and Ethical Considerations: Although glowing plants present exciting
                            scientific and practical prospects,
                            their development and use also face challenges. Ensuring that gene insertion does not
                            affect the plant's normal growth
                            or disrupt the ecosystem is a critical consideration. Moreover, ethical issues and
                            public acceptance must also be
                            carefully addressed. In summary, glowing plants represent an intriguing branch of
                            synthetic biology that showcases the
                            potential of biological engineering. They not only expand our understanding of plants
                            but also offer new possibilities
                            and challenges for the future of biotechnology.</p>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Applications of Plant Transient Transformation Technology</b>
                            </p>
                        </div>
                        <p>Plant Growth Cycle Records:</p>
                        <p>During the seedling cultivation period, we observed the following: 1. As shown in the raw
                            data in the Excel sheet below,
                            many leaves were still in the budding stage in the early period, so no growth records were
                            taken; only significant
                            growth was recorded later. 2. In the later growth records, some bottom leaves exhibited
                            slowed growth, fluctuations in
                            leaf length, and even wrinkling, which could be due to mild dehydration or the plant
                            sacrificing nutrients from the
                            lower leaves to support the growth of the upper leaves. 3. After more than four days of
                            recording, the growth slope of
                            newly emerged leaves was significantly steeper than that of the older leaves, indicating
                            that the new leaves grew much
                            faster than the older ones. This may have a beneficial effect on plant growth (apical
                            dominance).</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/21.png" alt="s6">
                            <figcaption>
                                <p>Figure 1: Growth Cycle Curve of Nicotiana Leaves</p>
                                <p>Leaf length is measured from the center of the plant to the furthest tip of the leaf.
                                </p>
                                <p>A: Growth stages of tobacco at different periods (a randomly selected tobacco
                                    seedling from the second batch, with
                                    records starting from 3 days after surface sterilization, 0 days after vernalization
                                    and sowing, 1 day after
                                    germination begins, and 10 days after transplanting).</p>
                                <p>B: Tobacco leaf growth curve.</p>
                                <p>C: Number and position of tobacco leaves.</p>
                            </figcaption>
                        </figure>
                        <p>
                            <object id="pdf" type="application/pdf"
                                data="https://static.igem.wiki/teams/5218/image/static/images/results/measurements.pdf"
                                class="pdf">
                                <a
                                    href="https://static.igem.wiki/teams/5218/image/static/images/results/measurements.pdf"></a>
                                <a
                                    href="https://static.igem.wiki/teams/5218/image/static/images/results/measurements.pdf">https://static.igem.wiki/teams/5218/image/static/images/results/measurements.pdf</a>
                            </object>
                            <!-- Fullscreen Button -->
                            <!-- <button onclick="toggleFullScreen()">
                                <p>Toggle Fullscreen</p>
                            </button> -->
                        </p>
                        <p>Due to the growth of the leaves, the gravity of the leaves will lower the height of the
                            plant, and the contact with the
                            plant during measurement may change the height of the plant, so the plant height here is
                            unstable and there is a large
                            fluctuation.</p>
                        <p>According to the chart, the plant height was generally slower in the early stage. The first
                            and second batches both had
                            large fluctuations in plant height between August 8 and August 14. Because the first batch
                            and the second batch were not
                            planted at the same time, the tobacco should be affected by environmental factors, and
                            therefore, the instability of
                            plant height makes plant height not a standard to measure the development of tobacco.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/22.png" alt="s6">
                            <figcaption>
                                <p>Figure 2: Growth Cycle Curve of Nicotiana Plant Height</p>
                                <p>Note: Plant height is measured as the maximum relative height from the stem above the
                                    soil surface to the leaf.</p>
                                <p>A: Average growth curve of tobacco plant height (three plants were randomly selected
                                    from each of the first and
                                    second batches of seedlings, and the average values were used for plotting).</p>
                                <p>B: Tobacco photos and schematic diagram of plant height measurement.</p>
                            </figcaption>
                        </figure>
                        <p>The Impact of Seedling Stage on Experimental Results:</p>
                        <p>In plant experiments, we typically use Agrobacterium-mediated transient transformation of
                            tobacco leaves for rapid
                            functional validation. When infecting leaves, older leaves are generally not selected under
                            non-specific conditions for
                            the following reasons:</p>
                        <p>1. Cell Activity: Older leaves have lower cell activity and reduced metabolic activity, which
                            are not conducive to the
                            expression of foreign genes and reduce transformation efficiency.</p>
                        <p>2. Tissue Structure: Older leaves have thicker cell walls and a harder texture, which may
                            hinder the transfer and
                            absorption of foreign DNA, reducing transformation efficiency.</p>
                        <p>3. Health Condition: Older leaves may harbor more pathogens or be more affected by
                            environmental stress, making them
                            less healthy than younger leaves, which could impact the accuracy of experimental results.
                        </p>
                        <p>4. Physiological State: Younger leaves are in a physiological state more suitable for gene
                            expression and protein
                            synthesis, enhancing the effectiveness of transient expression.</p>
                        <p>Therefore, selecting younger leaves for transient transformation experiments in tobacco can
                            improve transformation
                            efficiency and gene expression levels, ensuring the reliability and accuracy of the
                            experimental results.</p>
                        <p>To determine the optimal injection period for younger leaves, our data analysis indicated
                            that around the third week
                            (approximately 16 days) after transplanting is suitable for preparing infection solutions
                            and conducting bacterial
                            culture shaking. We then performed transient injections at various growth stages of the
                            tobacco seedlings, finding
                            notable differences between early-stage new leaves and mature leaves (26 days after
                            germination) at the injection stage.</p>
                        <p>Compared to younger leaves, mature tobacco leaves are fully expanded, have smoother surfaces,
                            and clearly defined veins
                            with fewer small veins, allowing a single injection to spread the infection solution over a
                            larger area, often covering
                            more than one-third of the leaf area, thereby reducing the number of injections needed. In
                            contrast, immature leaves are
                            not fully smooth, sometimes exhibiting uneven surfaces and numerous fine veins, making
                            injection more challenging and
                            resulting in a smaller spread area per injection, often not exceeding a 2 cm diameter
                            circle. This necessitates multiple
                            injections. Under these conditions, using overly young leaves for injection, such as when
                            comparing luminescence
                            differences, causes significant damage to the leaves and greatly affects the accuracy and
                            stability of functional
                            validation results.</p>
                        <p>Therefore, it is advisable to schedule bacterial culture shaking and infection solution
                            preparation around the third
                            week (approximately 16 days) after transplanting, with infection and phenotypic observation
                            conducted one month from the
                            initial sterilization treatment.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/23.png" alt="s6">
                            <figcaption>
                                <p>Figure 3: Visual Effects of Injections in Mature vs. Immature Leaves</p>
                                <p>A: Visual effect of injections in mature leaves at the correct stage. Injections are
                                    easy, with large and even water
                                    spread, causing minimal operational damage.</p>
                                <p>B: Visual effect of injections in immature leaves at an early stage. Water spread is
                                    small, the operation is
                                    difficult, and the damage is significant.</p>
                            </figcaption>
                        </figure>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Application of Synthetic Biology in Our Project</b>
                            </p>
                        </div>
                        <p>Our project aims to create glowing plant pets that heal the human spirit through luminous
                            plants. By enhancing plant
                            luminescence intensity based on fungal bioluminescence pathways, we achieve visibility at
                            night. We design biosynthetic
                            pathways within plants to significantly increase the production of caffeic acid, thereby
                            enhancing overall
                            bioluminescence. Beyond this, we combine dark therapy on a mental health level with glowing
                            plants to soothe the human
                            mind and alleviate stress.</p>
                        <p>Use of Plant Chassis (Glowing Plants):</p>
                        <p>The plant chassis offers unique advantages over microbial chassis in terms of cell structure,
                            product affinity, safety,
                            and storage. The development of plant tissue culture systems, such as hairy root cultures,
                            has accelerated the
                            commercialization and industrialization of synthetic biology in plant-derived natural
                            products.</p>
                        <p>Cell Structure: Plants have a finely compartmentalized cellular structure that supports the
                            proper functioning of
                            membrane-localized exogenous enzymes, particularly cytochrome P450 (CYP450) enzymes.</p>
                        <p>Species Compatibility: Plant chassis are better suited than microbial chassis for
                            expressing exogenous genes from
                            plant species. Natural product synthesis in plants often involves multiple biochemical
                            reactions and active enzymes,
                            some of which are absent in microbial chassis, requiring additional genes and hindering the
                            achievement of target
                            products.</p>
                        <p>Post-Translational Modification: Plant chassis possess comprehensive post-translational
                            modification systems that can
                            structurally modify natural products, ensuring the functionality of heterologous proteins.
                        </p>
                        <p>Cost Reduction: By synthesizing and accumulating natural products within the plant chassis,
                            costs associated with
                            cultivation, transportation, and preservation can be significantly reduced.</p>
                        <p>Stable Gene Integration: Plant chassis can integrate exogenous genes into the host genome
                            more stably, allowing these
                            genes to be inherited by the next generation. In contrast, prokaryotic chassis often rely on
                            recombinant plasmids that
                            exist independently of the host genome and can be lost during culture, leading to the loss
                            of exogenous genes.</p>
                        <p>Bioluminescence Mechanism in Synthetic Biology:</p>
                        <p>Bioluminescence is a widespread natural phenomenon observed in approximately 10,000 species.
                            It refers to the phenomenon
                            where living organisms emit light, or extracts of these organisms emit light in the
                            laboratory. This luminescence does
                            not rely on the absorption of light by organisms but is a special type of chemiluminescence
                            with nearly 100% efficiency
                            in converting chemical energy into light energy. It is also a form of oxidative
                            luminescence. The general mechanism of
                            bioluminescence involves chemical substances synthesized by cells that, under the action of
                            a special enzyme, convert
                            chemical energy into light energy.</p>
                        <p>Recently, a bioluminescence pathway using caffeic acid as a substrate was discovered in
                            fungi, producing green
                            fluorescence at a wavelength of approximately 520 nm. This pathway includes four key steps:
                        </p>
                        <p>1. HispS Enzyme: Uses ATP to convert caffeic acid into hispidin.</p>
                        <p>2. H3H Enzyme: Hydroxylates hispidin to form 3-hydroxyhispidin (fungal luciferin).</p>
                        <p>3. Luciferase: Fungal luciferin is oxidized by oxygen, producing an excited high-energy
                            intermediate, emitting green
                            fluorescence at 520 nm, along with carbon dioxide and caffeoylpyruvate.</p>
                        <p>4. CPH Enzyme: Hydrolyzes caffeoylpyruvate into pyruvate and caffeic acid, re-entering the
                            cycle [1].</p>
                        <p>Rapid Screening and Validation of Function Based on Luminescence Signal:</p>
                        <p>The main objective of this study is to screen genes related to caffeic acid biosynthesis and
                            introduce them into
                            tobacco, using genetic engineering to create a new type of bioluminescent plant. We expect
                            these experiments to
                            significantly enhance the efficiency of caffeic acid synthesis, thereby strengthening the
                            bioluminescent characteristics
                            of plants. Based on the principles of synthetic biology and plant synthetic biology, we
                            optimized the caffeic acid
                            biosynthesis pathway by replacing the key catalytic enzymes RtTAL and AtC3H. Through
                            transient injection at the optimal
                            stage of plant growth, we successfully validated that these modifications can enhance plant
                            bioluminescence.
                            Additionally, based on literature, we identified the GRF3 gene, which regulates plant
                            responses to formaldehyde, and
                            utilized its promoter to modify key catalytic enzyme expression modules. By integrating this
                            into the caffeic acid
                            synthesis pathway, the system activates key catalytic enzymes when formaldehyde levels are
                            high in the environment,
                            leading to increased caffeic acid synthesis and enhanced luminescence. This work aims to
                            create visually detectable
                            bioluminescent plants that serve not only as comforting “pets” but also as innovative
                            biological indicators for
                            environmental monitoring, particularly for detecting harmful gases such as formaldehyde.</p>
                        <figure>
                            <img class="text_img"
                                src="https://static.igem.wiki/teams/5218/image/static/images/results/24.png" alt="s6">
                            <figcaption>
                                <p>Figure 4: Functional Validation of Wet Lab Experiments</p>
                            </figcaption>
                        </figure>
                    </div>
                </section>

                <section id="sensor">
                    <h3>Plant Luminescent Sensors</h3>
                    <div style="text-align: left;">
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Construction of Plant Luminescent Sensing Module (Formaldehyde
                                    Response)</b>
                            </p>
                        </div>
                        <p>Based on the principles of plant synthetic biology, we observed the plant growth cycle,
                            expanded the use of
                            Agrobacterium-mediated transient transformation of tobacco leaves, and studied how plants
                            respond to formaldehyde,
                            utilizing this response to enhance luminescence intensity.</p>
                        <p>Studies have reported that the expression of the At14-3-3PSI (At14PSI/GRF3) gene in
                            Arabidopsis is rapidly
                            upregulated under formaldehyde stress. The knockout or reduction of AtGRF3, AtMDH1, or AtGS1
                            genes lowers the
                            plant's ability to absorb and resist HCHO. However, overexpression of AtGS1 and AtMDH1 genes
                            in the At14-3-3 psi
                            mutant restored Arabidopsis's uptake and resistance to HCHO. This indicates that AtGS1 and
                            AtMDH1 are indeed
                            indispensable for the metabolism of HCHO in Arabidopsis [2].</p>
                        <p>We designed primers to clone the promoter sequence of the AtGRF3 gene and replaced the
                            promoters of key catalytic
                            enzyme genes in the caffeic acid synthesis pathway. When formaldehyde concentration in the
                            plant increases, the
                            promoter responds and regulates caffeic acid synthesis, increasing substrate levels and
                            enhancing luminescence
                            intensity.</p>
                        <div style="text-align: left; color: #a8f900;">
                            <p><b style="color: #a8f900;">Application and Exploration of Plant Luminescent Sensors</b>
                            </p>
                        </div>
                        <p>Our luminescent plants that respond to formaldehyde can be considered plant-based biosensors.
                            Plant-based biosensors
                            have significant advantages in detecting soil and water pollution, as they can absorb
                            chemicals from the surrounding
                            environment as they grow naturally and exhibit morphological changes. This makes them more
                            convenient and efficient
                            than existing electronic detection devices and more responsive to low-concentration
                            pollutants. Additionally, plant
                            biosensors have the added benefit of not affecting the natural environment while detecting
                            pollutants and
                            metabolizing harmful substances.</p>
                        <p>Once the appropriate indicators are determined, ordinary people can visually observe their
                            shape and quickly detect
                            pollutants. In the future, we can continue to use the growth cycle data of Nicotiana
                            benthamiana for transient
                            functional validation of other genes to develop more application scenarios.</p>
                        <p>This innovative bioluminescent plant, as an ornamental plant, can also serve as a "pet" that
                            soothes the soul. By
                            increasing the concentration of caffeic acid, we aim for the bioluminescence to be visible
                            to the naked eye and felt
                            by the heart. By integrating our glowing plants into dark therapy, a method used in
                            psychological treatment, we hope
                            to relieve people's stress, release emotions, ease tension, and even help alleviate
                            depression. Through this
                            research, our goal is to promote the practical application of biotechnology in everyday
                            life.</p>
                        <p>In the rapidly developing field of synthetic biology, there is an urgent need for robust
                            bioethical frameworks to
                            address emerging challenges. Due to its rapid growth, the ethical guidelines related to
                            synthetic biology are not
                            always comprehensive. This reality highlights the necessity for individuals working in this
                            field to take
                            responsibility for ensuring their progress adheres to principles of ethics, responsibility,
                            and genuine benefit to
                            the world.</p>
                        <p>Synthetic biology possesses extraordinary capabilities, enabling highly complex and modular
                            systems to address some
                            of humanity's most pressing issues through the standardization of biological components. For
                            example, in our
                            project, using plant synthetic biology, we successfully validated bioluminescent plants
                            capable of detecting
                            formaldehyde without harmful chemical reagents, thereby reducing environmental pollution and
                            aligning with
                            sustainable development principles.</p>
                        <p>However, the responsibility to assess the impact of our creations should not rest solely with
                            governments and
                            ethicists. While they are valuable resources, we, as individuals, have the duty to analyze
                            our creations from a
                            bioethical perspective and uphold the principles of scientific integrity.</p>
                    </div>
                </section>

                <section id="references">
                    <h3>References</h3>
                    <div style="text-align: left;">
                        <p>[1] Mitiouchkina, T., Mishin, A.S., Somermeyer, L.G. et al. Plants with genetically encoded
                            autoluminescence. Nat
                            Biotechnol 38, 944–946 (2020).</p>
                        <p>[2] Xing Zhao, Xueting Yang, Yunfang Li, Hongjuan Nian, Kunzhi Li. (2023) 14-3-3 proteins
                            regulate the HCHO stress
                            response by interacting with AtMDH1 and AtGS1 in tobacco and Arabidopsis. Journal of
                            Hazardous Materials, 458, 132036</p>
                    </div>
                </section>

            </div>
        </div>
    </section>

    <!-- Collection Section -->
    <section class="collection" style="background-color: rgba(30, 30, 30);">
        <div class="partners">
            <img class="partners__image"
                src="https://static.igem.wiki/teams/5218/image/static/images/sponsor/all.png" />
        </div>

        <div class="contacts">
            <p class="contacts__description">Address: BGI Center, No. 9 Yunhua Road, Yantian District,
                Shenzhen, 518083,
                China</p>
            <p class="contacts__description">Phone: +86 13101793873 (Weekdays: 8:00-17:00 Weekends: Closed)
                E-mail:
                chenlichuan@genomics.cn</p>
        </div>

        <div class="copyright">
            <p class="copyright__text">
                ©2024 Content on this site is licensed under a
                <a class="copyright__link" href="https://creativecommons.org/licenses/by/4.0/">Creative Commons
                    Attribution 4.0 International license</a>
            </p>
            <p class="copyright__text">
                The repository used to create this website is available at
                <a class="copyright__link" href="">gitlab.igem.org/2024/bgi-mammothedu-south</a>
            </p>
        </div>
    </section>

    <!-- JavaScript -->

    <!-- Loader -->
    <script>
        // Remove the preloader when the page is fully loaded
        window.onload = function () {
            const preloader = document.getElementById('preloader');
            preloader.style.opacity = '0'; // Start fading out the preloader
            setTimeout(() => {
                preloader.style.display = 'none'; // Hide the preloader completely
            }, 500); // Wait for the fade-out to complete
        };
    </script>

    <!-- Image slider -->
    <script>
        let currentSlide = 0;
        const slides = [
            "linear-gradient(to bottom, rgba(0, 0, 0, 0.7), rgba(255, 255, 255, 0)), url('https://static.igem.wiki/teams/5218/image/static/images/results/1.jpg')",
            "linear-gradient(to bottom, rgba(0, 0, 0, 0.7), rgba(255, 255, 255, 0)), url('https://static.igem.wiki/teams/5218/image/static/images/results/2.jpg')",
            "linear-gradient(to bottom, rgba(0, 0, 0, 0.7), rgba(255, 255, 255, 0)), url('https://static.igem.wiki/teams/5218/image/static/images/results/3.jpg')"
        ];
        const dots = document.querySelectorAll('.carousel-nav span');
        const introSection = document.getElementById('intro-section');

        function showSlide(index) {
            dots[currentSlide].classList.remove('active');
            currentSlide = index;
            introSection.style.backgroundImage = slides[currentSlide]; // Apply the gradient and image
            dots[currentSlide].classList.add('active');
        }

        // Make sure the first slide is loaded when the page loads
        document.addEventListener('DOMContentLoaded', function () {
            // Preload images
            slides.forEach(function (slide) {
                const img = new Image();
                img.src = slide.match(/url\('(.*)'\)/)[1]; // Extract the URL from the background-image
            });

            showSlide(0); // Show the first slide immediately after preloading starts
        });

        setInterval(() => {
            const nextSlide = (currentSlide + 1) % slides.length;
            showSlide(nextSlide);
        }, 2000); // Change every 2 seconds
    </script>

    <!-- ToC -->
    <script>
        // Smooth scroll when clicking ToC links
        document.querySelectorAll('.toc a').forEach(anchor => {
            anchor.addEventListener('click', function (e) {
                e.preventDefault();

                document.querySelector(this.getAttribute('href')).scrollIntoView({
                    behavior: 'smooth'
                });
            });
        });
    </script>

    <!-- pdf -->
    <script>
        function toggleFullScreen() {
            var elem = document.getElementById("pdf");

            if (!document.fullscreenElement) {
                if (elem.requestFullscreen) {
                    elem.requestFullscreen();
                } else if (elem.mozRequestFullScreen) { // Firefox
                    elem.mozRequestFullScreen();
                } else if (elem.webkitRequestFullscreen) { // Chrome, Safari and Opera
                    elem.webkitRequestFullscreen();
                } else if (elem.msRequestFullscreen) { // IE/Edge
                    elem.msRequestFullscreen();
                }
            } else {
                if (document.exitFullscreen) {
                    document.exitFullscreen();
                } else if (document.mozCancelFullScreen) { // Firefox
                    document.mozCancelFullScreen();
                } else if (document.webkitExitFullscreen) { // Chrome, Safari and Opera
                    document.webkitExitFullscreen();
                } else if (document.msExitFullscreen) { // IE/Edge
                    document.msExitFullscreen();
                }
            }
        }
    </script>

    <!-- Header -->
    <script src="./static/js/header.js"></script>

</body>

</html>